jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -451,7 +455,7 @@ def overlay_regions(self):
         Access the regions objects corresponding to the current settings
         """
 
-        callable_kwargs = {k: getattr(self, k)
+        callable_kwargs = {k: float(getattr(self, k))

```

Comment:
I found some cases where the traitlet for PA was being interpreted as a string, which causes problems when a float PA offset is subtracted. Here I ensure we're only giving these properties as floats.